### ISSUE TITLE
Release.sh has misconfigured paths?

### ISSUE THREAD

**Issue by foxclore** [GrapheneOS/os-issue-tracker#849]
After successful build I tried generating release, and release.sh fails with the following errors:

grep: /vendor/google_devices/sunfish/vendor-board-info.txt: No such file or directory
grep: /vendor/google_devices/sunfish/vendor-board-info.txt: No such file or directory
Traceback (most recent call last):
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/internal/stdlib/runpy.py", line 174, in _run_module_as_main
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/internal/stdlib/runpy.py", line 72, in _run_code
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/__main__.py", line 12, in <module>
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/internal/stdlib/runpy.py", line 174, in _run_module_as_main
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/internal/stdlib/runpy.py", line 72, in _run_code
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/sign_target_files_apks.py", line 1401, in <module>
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/sign_target_files_apks.py", line 1351, in main
  File "/mnt/volume2/grapheneos-SP1A.211105.004.2021112404/out/release-sunfish-eng.root.20211204.204935/bin/sign_target_files_apks/internal/stdlib/zipfile.py", line 756, in __init__
IOError: [Errno 2] No such file or directory: '/obj/PACKAGING/target_files_intermediates/sunfish-target_files-eng.root.20211204.204935.zip'

Somewhy it tries to reach paths from the fs root (/). Grep error can be resolved if you copy vendor-board-info.txt to /
Seems release.sh cant correctly find current directory?

**Comment by thestinger** [comment]
It expects the `$OUT` and `$BUILD_NUMBER` variables to be set. Could add a check for `$OUT` similar to build number.

**Comment by foxclore** [comment]
Hm, but there was nothing about setting $OUT in [build.grap](https://grapheneos.org/build#building), what kind of path it expects it to be? just the out directory?

**Comment by thestinger** [comment]
Both are set by choosecombo.

**Comment by foxclore** [comment]
Oh okay good. I did building and creating release in two different days, therefore ran choosecombo only once. Would be nice to have a check in release.sh! Thanks, issue resolved. Good luck and have a great day!

**Comment by thestinger** [comment]
Check is added now.
